src/components/notifications/PageNotifications.vue

```diff
@@ -75,8 +110,11 @@ export default {
         }
       },
       /* istanbul ignore next */
-      update(data) {
-        return data.notifications
+      update(result) {
+        this.dataLoaded = true
+        // assume that when the full page got loaded, that there is more
+        this.moreAvailable = result.notifications.length % 20 === 0
```

Comment:
Can you wrap this with `( ... )` , makes it more clear that it's an evaluation (unless the linter removes this)

Reply:
do you know the suggestion function in GitHub?
![image](https://user-images.githubusercontent.com/5869273/83763935-e5189c80-a63e-11ea-895e-10e3a69bcadf.png)

Reply:
```suggestion
        this.moreAvailable = (result.notifications.length % 20 === 0)
```